pkg/cleaner/cleaner.go

```diff
@@ -26,7 +25,7 @@ func Run(gitHome string, nsLister k8s.NamespaceWatcher) error {
 		if event.Type == "DELETED" {
 			namespace := event.Object.(*api.Namespace)
```

Comment:
@smothiki you should check this type assertion, and if it wasn't an `*api.Namespace`, then don't continue.